eve/main.yml

```diff
@@ -93,8 +97,32 @@ stages:
     - TriggerStages:
         name: trigger all the tests
         stage_names:
+        - build-e2e
         - helm_test_kube_1.9.6
 
+  build-e2e:
+    worker: *pod
+    steps: &build_e2e
+    - Git: *git_pull
+    - ShellCommand:
+        name: Private Registry Login
+        command: >
+          docker login
+          -u '%(secret:private_registry_username)s'
+          -p '%(secret:private_registry_password)s'
+          '%(secret:private_registry_url)s'
+    - SetProperty: *docker_image_name
+    - ShellCommand:
+        name: build end2end image
+        command: make DOCKER_IMAGE_NAME='%(prop:docker_image_name)s' DOCKER_IMAGE_TAG='%(prop:commit_short_revision)s' container-image
```

Comment:
cosmetic: could use > notation to split this line into multiple shorter lines

Reply:
Good point, done!
Thanks